app/components/github-repo.js

```diff
@@ -0,0 +1,22 @@
+import Ember from 'ember';
+import recordsList from 'code-corps-ember/utils/records-list';
+
+const {
+  computed,
+  Component,
+  get
+} = Ember;
+
+export default Component.extend({
+  classNames: ['github-repo'],
+  tagName: 'li',
+
+  isConnected: computed('githubRepo', 'project.projectGithubRepos.@each', function() {
+    let githubRepo = get(this, 'githubRepo');
+    let project = get(this, 'project');
+    let projectGithubRepos = get(this, 'project.projectGithubRepos');
+    if (projectGithubRepos) {
+      return recordsList.includes(projectGithubRepos, githubRepo);
+    }
```

Comment:
Not sure how to get this to work. Get the following:

> There is no belongsTo relationship named 'github-repo' on a model of modelClass 'project-github-repo'

...which is obviously not true.